corehq/apps/reports/dispatcher.py

```diff
@@ -176,6 +172,7 @@ def dispatch(
             and report_class.allow_access(request)
         ):
             try:
+                report_kwargs = kwargs.copy()
```

Comment:
nit: why is this in the `try` block?  IMO it's preferable to keep those blocks as narrow as possible.

Reply:
Good point. My intention was just to move it inside the `if` block, because that's the only place it's used.

![image](https://user-images.githubusercontent.com/708421/151962741-3960c833-3f44-4516-9825-3ef4ce3f48ce.png)

"Do. Or do not. There is no try." 2348409e86c